### PR TITLE
Fixing Travis CI configuration

### PR DESCRIPTION
**Pull request by evertonfraga**
This PR aims to fix Travis' configurations.

- [x] Update travis to nodejs v7.x latest
- [x] Update .travis.yml file to latest testing commands
- [x] Travis reporting initial status as "build is in progress"
- [x] CI ran after PR creation.
- [x] CI ran after adding any commits to the same branch.

### CODE REVIEW

**Comment by evertonfraga** [comment]
@frozeman ready for review.

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,13 @@
 language: node_js
 node_js:
-  - "4.1"
-  - "4.0"
+  - "7"
+  - "8"
```
Comment (frozeman) [inline .travis.yml]: I also want to keep an older node version to test it there. Can you please add at least 5 and 6?

**Comment by frozeman** [comment]
Closing this, as i fixed it in the branch directly

**Comment by evertonfraga** [comment]
👍🏼

**Pull request closed without merging**